pkgs/applications/networking/flexget/default.nix

```diff
@@ -36,22 +36,25 @@ with python'.pkgs;
 
 buildPythonApplication rec {
   pname = "FlexGet";
-  version = "2.17.14";
+  version = "2.17.20";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1wh12nspjzsgb0a7qp67s4k8wssbhhf500s8x8mx2smb1mgy4xzz";
+    sha256 = "1hw16rn3wsc3adkx5ja16cxc9ic7jv2hhkcbxfb7wkym5r4gk7m0";
   };
 
   postPatch = ''
     # remove dependency constraints
     sed 's/==\([0-9]\.\?\)\+//' -i requirements.txt
+    # Set python to 3.6
+    sed -i -e 's/py27/py36/g' setup.cfg
```

Comment:
You're sure this is necessary?

Reply:
it is, otherwise it defaults to python 2.7.

Reply:
It would probably not if we didn't have `python27` anywhere.

Reply:
cherrypy won't work with python27 anymore. That's the whole reason for going for python 36